src/screens/home/AnnouncementDetailsScreen.tsx

```diff
@@ -134,7 +134,11 @@ export default function AnnouncementDetailScreen({
         {announcementItem?.callToAction ? (
           <WhiteButton
             style={{ height: 56, marginBottom: 40, marginTop: 40 }}
-            label="Call to Action"
+            label={
+              announcementItem?.callToActionTitle === ''
```

Comment:
This could be `announcementItem?.callToActionTitle ?? 'Attend'`.

Unless you want the label to be blank if `announcementItem?.callToActionTitle` evaluates to `null` or `undefined`.

Reply:
When ```callToActionTitle``` is not set when creating an announcement, it will be ```=== ""```.  
```announcementItem?.callToActionTitle ?? 'Attend'``` causes the button to show without a label when it is not specified when the announcement is created.

Reply:
Ah I see. 

Regardless the logic can be simplified a bit: `announcementItem?.callToActionTitle ? announcementItem?.callToActionTitle : 'Attend'`

Reply:
Thank you for this. I have implemented your suggestion.